addon/components/custom-subsidy-form-fields/estimated-cost-table/edit.js

```diff
@@ -348,13 +348,23 @@ export default class CustomSubsidyFormFieldsEstimatedCostTableEditComponent exte
       this.storeOptions.sourceGraph
     );
 
-    const validCosts = entries.filter(entry => parseInt(entry.object.value) > 0 );
-    if(!validCosts.length){
+    const inValidCosts = entries.filter(entry => isNaN(parseInt(entry.object.value)));
+    if(inValidCosts.length) {
+      this.errors.pushObject({
+        message: 'Eén van de velden werd niet correct ingevuld.'
+      });
+      this.updateTripleObject(this.estimatedCostTableSubject, validEstimatedCostTable, null);
+    }
+
+    const positiveCosts = entries.filter(entry => parseInt(entry.object.value) > 0 );
+    if(!positiveCosts.length){
       this.errors.pushObject({
         message: 'Mintens één kosten veld moet een waarde groter dan 0 bevatten.'
       });
       this.updateTripleObject(this.estimatedCostTableSubject, validEstimatedCostTable, null);
-    } else {
+    }
+
+    if(positiveCosts.length && !inValidCosts.length) {
```

Comment:
if, else if, else doesnt work here as we want both positiveCost & inValidCost to be correct